nirtorch/graph.py

```diff
@@ -21,6 +21,9 @@ def named_modules_map(
     """
     modules_map = {}
     for name, mod in model.named_modules():
+        # Ignore sequential modules
+        if isinstance(mod, nn.Sequential):
```

Comment:
This looks like a hack. Why is a sequential model skipped?

Reply:
Because any sequential model is basically a wrapper that we can safely ignore in the graph. Concretely, instead of going `A -> (Sequential_input -> B -> Sequential_output) -> C`, we can say `A -> B -> C`

Reply:
IMHO, If someone decides to structure their model this way, the graph should ideally reflect this structure, how ever trivial this is.